geps/gep-1897.md

```diff
@@ -0,0 +1,172 @@
+# GEP-1897: TLS from Gateway to Backend for ingress
+
+* Issue: [#1897](https://github.com/kubernetes-sigs/gateway-api/issues/1897)
+* Status: Provisional
+
+## TLDR
+
+This document specifically addresses the topic of conveying HTTPS from the Gateway
+dataplane to the backend (backend TLS termination), and intends to satisfy the single
+use case “As a client implementation of Gateway API, I need to know how to connect to
+a backend pod that has its own certificate”. TLS configuration can be a nebulous topic,
+so in order to drive resolution this GEP focuses only on this single piece of functionality.
+
+## Immediate Goals
+
+1. The solution must satisfy the following use case: the backend pod has its own
+certificate and the gateway implementation client needs to know how to connect to the
+backend pod. (Use case #4 in [Gateway API TLS Use Cases](#references))
+2. In terms of the Gateway API personas, only the application developer persona in this
+solution. The application developer should control the gateway to backend TLS settings,
+not the cluster operator, as requiring a cluster operator to manage certificate renewals
+and revocations would be extremely cumbersome.
+3. The solution should consider client certificate settings used in the TLS handshake **from
```

Comment:
sorry to nitpick but version and cipher suites are TLS handshake properties, not "certificate settings"

Reply:
Fixed